client/reconfiguration/test/poll_based_state_client_test.cpp

```diff
@@ -130,7 +130,8 @@ class ClientApiTestFixture : public ::testing::Test {
     auto* orig_msg = reinterpret_cast<const bftEngine::ClientRequestMsgHeader*>(msg.data.data());
     if (!(orig_msg->flags & RECONFIG_FLAG)) {
       auto reply = createReply(msg);
-      client_receiver->onNewMessage(msg.destination.val, reinterpret_cast<const char*>(reply.data()), reply.size());
+      client_receiver->onNewMessage(
```

Comment:
No need to pass the same value twice - let the default value take place. Everywhere in this file.

Reply:
done.